e2e/cypress/tests/integration/system_console/true_up_review_spec.js

```diff
@@ -0,0 +1,182 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import moment from 'moment-timezone';
+
+describe('System Console > Site Statistics > True Up Review', () => {
```

Comment:
Could we also add a test to not show the true up review if telemetry is enabled?

Reply:
Definitely. I think there was one, but the name was in a bade state. I've updated the name and added an overwrite for the config since we're getting it differently now.